keyboards/protokeeb/rev1/config.h

```diff
@@ -0,0 +1,50 @@
+/* Protokeeb Copyright 2024 A-Tech Officials (@atechofficials)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. 
+ * If not, see <https://www.gnu.org/licenses/gpl-3.0.html>.
+ */
+
+#pragma once
+
+// Define the pin connected to the encoder push button
+#define ENCODER_BUTTON_PIN GP14
+
+// Define the row and column for the encoder button
+#define ENCODER_BUTTON_ROW 0
+#define ENCODER_BUTTON_COL 4
+
+/* Peripheral used */
+#define WS2812_PIO_USE_PIO1
+
+/* RGB lighting effects and animations settings */
+#define RGBLIGHT_EFFECT_BREATHE_MAX 127
+#define RGBLIGHT_EFFECT_CHRISTMAS_INTERVAL 40
+#define RGBLIGHT_EFFECT_CHRISTMAS_STEP 2
+#define RGBLIGHT_EFFECT_KNIGHT_LED_NUM 4
+#define RGBLIGHT_EFFECT_KNIGHT_LENGTH 3
+#define RGBLIGHT_EFFECT_KNIGHT_OFFSET 0
+#define RGBLIGHT_EFFECT_SWIRL_RANGE 255
+#define RGBLIGHT_EFFECT_SNAKE_LENGTH 4
+#define RGBLIGHT_EFFECT_TWINKLE_LIFE 200
+#define RGBLIGHT_EFFECT_TWINKLE_PROBABILITY (1/127)
+
+/* Double tap reset bootloader entry */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+/* Bootloader timeout window in milli-seconds */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
+
+/* LED to blink when entering bootloader mode */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
+
```

Comment:
There is no LED connected to this pin in this keyboard's schematic. I advise against copy/pasting code.
![image](https://github.com/user-attachments/assets/8b21aeca-320b-4cb9-b3a3-44befc24cc84)
```suggestion
```

Reply:
If you have closely examined the PCB files for this keyboard, you would not accuse me of blindly copying and pasting code from other keyboard firmware developers. I am using a Raspberry Pi Pico (RP2040) development board as the main controller for this keyboard, and the built-in LED of the Raspberry Pi Pico is connected to GPIO25. Therefore, if I am using this built-in LED to visually indicate when the keyboard enters in bootloader mode, what is wrong with that?

Reply:
Ah, fair enough. Wonder if it would be wise to add a small comment like
```c
// Pi Pico's on board LED
#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
```
So there is not confusion for others using a different development board.

Reply:
Currently, the PCB of the Protokeeb Rev1 keyboard is compatible with the Raspberry Pi Pico (RP2040) and the Waveshare RP2040-Plus development boards. Any other RP2040 development boards will not be compatible unless they have the same footprint and pin assignments as the Raspberry Pi Pico (RP2040). Additionally, I will make sure to include that comment you suggested.